Gruntfile.js

```diff
@@ -133,8 +133,8 @@ module.exports = function (grunt) {
                     },
                     {
                         expand: true,
-                        cwd: 'node_modules/css-layout/src/',
-                        src: ['Layout.js'],
+                        cwd: 'node_modules/css-layout/dist/',
```

Comment:
I know this is a pre-existing problem but can the d3, css-layout and d3fc dependencies be collapsed into one block?